backend/app/wallet/__init__.py

```diff
@@ -176,6 +176,15 @@ def cancel_transaction(txn: str, request: Request, login=Depends(login_state)):
     return Response(None, status_code=201)
 
 
+@router.post("/process-pending-transfers")
+def process_transfers():
```

Comment:
Bart has added a task scheduler, we need to switch to that.

Reply:
I have added something to `worker.py` @barthalion what do I need to add to, to ensure that a dramatiq actor is triggered?

Reply:
You need to call `.send()` method like at https://github.com/flathub/website/blob/main/backend/app/main.py#L74. The job will be picked up by a worker when free. This is tricky to test in CI, I ended up adding `time.sleep` to at least one test to account for async task execution.

Reply:
And as we discussed previously, if you need to run this on schedule, we call `/update` every hour via cronjob, but I can add a separate trigger for e.g. `/update/process-pending-transfers`. No need to add auth, I will just lock access to anything /update on nginx level.

Reply:
Aah I figured that dramatiq *was* that scheduler.  I think that hourly is plenty so I'll tack it into the `/update` actor instead.  Thanks for explaining.

Reply:
OK, so the dramatiq worker doesn't actually have the DB session enabled, and it looks more sensible for me to re-attach the way I was doing it to `/update/process-pending-transfers` as you suggest, rather than putting it into dramatiq, do you want me to document the need to ping that endpoint anywhere?

Reply:
It's backed by redis as a DB

Reply:
> the dramatiq worker doesn't actually have the DB session enabled

Haven't looked into how we use sqlalchemy/psycopg, but as long as it's initialized somewhere in the imported modules, it should work fine

Reply:
Sadly we use the FastAPI DBSessionMiddleware stuff, which I don't think can be initialised outside of FastAPI.  I seriously don't think we're going to have enough transfers to need the processing of them to be moved out to dramatiq but we can always do that later if it's worth it.

Reply:
I've made a README note for this.